tbot_contrib/swupdate/swupdate_script.py

```diff
@@ -0,0 +1,97 @@
+import asyncio
+import json
+import requests
+import websockets
+import argparse
+
+
+class SWUpdater:
+    "" " Python helper class for SWUpdate " ""
+
+    url_upload = "http://{}:8080/upload"
+    url_status = "ws://{}:8080/ws"
+
+    def __init__(self, path_image: str, host_name: str):
+        self.__image = path_image
+        self.__host_name = host_name
```

Comment:
Names starting with `__` (double underscore) are reserved for Python itself.  You should prefer using a single underscore as a prefix for private variables.